data/controller_data.py

```diff
@@ -84,6 +84,12 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
+    'tags': ['console']},
```

Comment:
Can you fix the spacing of this please (indent by one).

Reply:
Done.